CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ Inspired from [Keep a Changelog](https://keepachangelog.com/en/1.0.0/)
 ## [Unreleased]
 ### Added
 - Added deprecation warnings for Index Template APIs ([654](https://github.com/opensearch-project/opensearch-js/pull/645))
+- Added Low-level APIs functions ([#649](https://github.com/opensearch-project/opensearch-js/pull/649))
```

Comment:
Nit: we should say which ones, I would say "Added `http.get`, `.post` ..." or something along these lines.